GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -703,5 +705,84 @@ private void ShowSeparatePushUrl(bool visible)
                 ? _labelUrlAsFetch.Text
                 : _labelUrlAsFetchPush.Text;
         }
+
+        public class ComboBox : System.Windows.Forms.ComboBox
```

Comment:
One class per file, please, i.e. move to `GitUI\UserControls`.

Reply:
Moved and renamed. It's no longer a subclass of FormRemotes. Not sure why I didn't do that to begin with since it's a general use Combobox replacement with case sensitive autocomplete.